docs/pages/guides/nextjs-starter-kit.mdx

```diff
@@ -0,0 +1,545 @@
+   place it within `.env.local` as `AUTH0_ISSUER_BASE_URL`.
+8. Add the following to the "Allowed Callback URLs" textarea:
+   `http://localhost:3000/api/auth/callback/auth0`.
+9. Add the following to the "Allowed Logout URLs" textarea:
+   `http://localhost:3000`.
+10. Scroll to the bottom and press "Save changes".
+
+<Figure>
+  <video autoPlay loop muted playsInline>
+    <source
+      src="/images/docs/next-js-starter-kit/auth0-auth.mp4"
+      type="video/mp4"
+    />
+  </video>
+</Figure>
+
+`.env.local` should now contain these three lines, along with anything
+previously there:
+
+```dotenv file=".env.local"
+AUTH0_CLIENT_ID=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+AUTH0_CLIENT_SECRET=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+AUTH0_ISSUER_BASE_URL=https://XXXXXXXXXXXXXXXXXX.com
+```
+
+Auth0 authentication is now set up! Next, [add yourself a user](#how-to-sign-up)
+to test out your authentication.
+
+### How to sign up - add yourself as a user [#how-to-sign-up]
+
+We haven't set up a database, so we're temporarily using the `/data` folder
+instead. Before any user can sign in, they need to be added to `/data/users.ts`.
+Navigate there and add your details, for example, if you're signing in with
+`yourname@example.com`:
+
+```ts file="data/users.ts"
+{
+  id: "yourname@example.com",
+  name: "Your Name",
+  avatar: "https://liveblocks.io/avatars/avatar-0.png",
+  groupIds: ["product", "engineering", "design"],
+},
+```
+
+### Ready to go
+
+The Next.js starter kit is now ready to use! After setting up authentication,
+make sure to restart the dev server to see your authentication in action.
+
+```
+npm run dev
+```
+
+## Assorted info
+
+### Async Fetching and Error Handling
+
+This starter kit makes extensive use of the following programming pattern for
+fetching async resources:
+
+```ts
+const { data, error } = await getDocument({
+  /* ... */
+});
+
+// An error has occured
+if (error) {
+  // { code: 400, message: "Document not found", suggestion: "Please check the URL is..." }
+  console.log(error);
+  return;
+}
+
+// Success, but result is empty
+if (!data) {
+  return;
+}
+
+// Success
+// { name: "my-document", id: "hIas7GuihgHF8Fhv8Sskg",  ... }
+console.log(data);
+```
+
+### Document functions
+
+Much of the starter kit's power is in the `/lib/server/documents` directory. The
+functions in these files allow you to edit your documents easily and return
+type-safe objects. For example in an API endpoint:
+
+```ts
+import { createDocument } from "../../lib/server";
+
+export default async function (req, res) {
+  // Create a new document
+  const { data, error } = await createDocument(req, res, {
+    name: "My document",
+    type: "whiteboard",
+    userId: "charlie.layne@example.com",
+  });
+  // ...
+}
+```
+
+These functions can also be leveraged on the client side. To do so, simply
+import from the `/lib/client` folder, and omit `res` and `res` instead. This
```

Comment:
```suggestion
import from the `/lib/client` folder, and omit `req` and `res` instead. This
```

I feel there's an issue with the part where we say 
> omit `res` and `res` instead


Shouldn't it be `req` and `res`? Otherwise, I just don't understand the meaning tbh. Maybe highlight the change or explain it?